pkg/cstor/cstor.go

```diff
@@ -134,6 +136,28 @@ func (p *Plugin) getServerAddress() string {
 
 // getMapiAddr return maya API server's ip address
 func (p *Plugin) getMapiAddr() string {
+	sclist, err := p.K8sClient.Services(operator).List(
```

Comment:
Will this check in all namespaces?

Reply:
No. It will check only in `openebs` namespace.
We are assuming that openebs is installed in `openebs` namespace only.

If user is having openebs installed in multiple or different namespace then it will fail.
We can make it configurable by adding a new parameter like `openebs_namespace`, to `volumesnapshotlocation`. This will work if openebs is installed in different namespace other than `openebs` or having multiple installation of openebs.